src/components/common/YourCompany/ChangeBusinessType.vue

```diff
@@ -405,9 +409,9 @@ export default class ChangeBusinessType extends Mixins(CommonMixin) {
     // reset name request
     this.setNameRequest({
       legalType: this.getEntitySnapshot?.businessInfo?.legalType,
-      legalName: this.getEntitySnapshot?.businessInfo?.legalName,
```

Comment:
The local "hacked" name request interface contained `legalName` aka Approved Name and sometimes Company Name. I pulled it out into its own store variable so that the name request object is un-sullied.